jwst/datamodels/make_header.py

```diff
@@ -323,7 +323,7 @@ def import_objects(package, line):
     local_variables = {}
     global_variables = {'__package__':package}
     try:
-        exec(line, global_variables, local_variables)
+        exec(line, global_variables, local_variables) # nosec
```

Comment:
This whole module does a bunch of source code parsing and executing imports found at the top of various pipeline modules.  It would be better to use `importlib` and `ast` for all of this if anyone want to take a crack at it in the future.